lib/consyncful/sync.rb

```diff
@@ -52,6 +60,12 @@ def load_all_models
       Rails.application.eager_load!
     end
 
+    def update_run(next_url)
```

Comment:
Extract method from `run` to reduce line count

Reply:
Neat.